### ISSUE TITLE
Blackstone Furnaces don't work at all

### ISSUE THREAD

**Issue by mapleexists** [mschae23/Blackstone-Tools#4]
Blackstone furnaces don't cook anything regardless of the fuel or the input. Everything about it works fine except for the fact that it doesn't start cooking once both items are put into the slots.

**Comment by mschae23** [comment]
Yes, that's a known issue. I don't know why they don't work, though. I made blackstone furnaces not craftable because of this.
